firstvoices/backend/admin/app_admin.py

```diff
@@ -35,3 +35,18 @@ class AppMembershipAdmin(BaseAdmin):
         "user",
         "role",
     )
+
+
+@admin.register(AppImportStatus)
+class AppImportStatusAdmin(BaseAdmin):
+    fields = (
+        "label",
+        "warnings",
+        "successful",
+    )
```

Comment:
Adding these three fields as [readonly_fields](https://docs.djangoproject.com/en/4.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.readonly_fields) might be good here to make sure we don't accidentally change these values in the admin UI.